jest.config.ts

```diff
@@ -0,0 +1,13 @@
+import type { Config } from '@jest/types';
+
+const config: Config.InitialOptions = {
+  preset: 'ts-jest',
+  testEnvironment: 'node',
+  setupFilesAfterEnv: ['./__tests__/setup.ts'],
+  testPathIgnorePatterns: [
+    '/node_modules/', // Ignore files in node_modules directory
+    '/__tests__/setup' // Ignore setup files within __tests__ directory
+  ]
```

Comment:
See https://jestjs.io/docs/configuration#testregex-string--arraystring

Reply:
Thanks for this insight 👍